src/EnergyPlus/WaterCoils.hh

```diff
@@ -65,20 +66,27 @@ struct EnergyPlusData;
 
 namespace WaterCoils {
 
+    enum class iCoilModel
+    {
+        Unassigned,
+        Simple,
+        Cooling,
+        Detailed,
```

Comment:
Never mind my earlier comment - my page wasn't refreshed.
But these enum names here should be "HeatingSimple" "CoolingSimple" and "CoolingDetailed", I'm fairly certain - haven't gone looking for their usage.